Cargo.toml

```diff
@@ -7,12 +7,20 @@ version = "0.7.0"
 authors = ["erick.tryzelaar@gmail.com"]
 license = "MIT/Apache-2.0"
 
+build = "build.rs"
+
 [features]
 unstable = []
+test-serde = ["serde", "serde_json", "serde_codegen"]
 
 [dependencies]
 rustc-serialize = "0.3.18"
 log = "0.3.5"
+serde = { version = "0.7", optional = true }
```

Comment:
`serde 0.8` is now officially a thing. Can we upgrade to it? I'm happy to help out if it turns out to be a lot of effort.